scripts/validate

```diff
@@ -17,5 +15,3 @@ echo Running validation
 echo Running: golangci-lint
 golangci-lint run
 
-echo Running: go fmt
-test -z "$(go fmt ${PACKAGES} | tee /dev/stderr)"
```

Comment:
How come this is removed ?

Reply:
`golangci-lint run` already runs this.

Reply:
https://github.com/rancher/backup-restore-operator/blob/master/.golangci.json#L10